llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -37,6 +37,13 @@ class RISCVExtension<string name, int major, int minor, string desc,
   bit Experimental = false;
 }
 
+// TODO: Maybe integrate into RISCVExtension class
```

Comment:
My first impression is could we integrate into RISCVExtension, and then saw this comment, so let give a try :P

Reply:
Updated.